### ISSUE TITLE
Need uploaded files count but cant find

### ISSUE THREAD

**Issue by CeRBeR666** [pqina/filepond#705]
Greetings friends,
I've been trying to find a solution for about 2 days, but I couldn't. As a last resort, I decided to write here.
I am using filepond and php (phalcon) in my last project. I didn't have any problems uploading files and I liked the filepond very much. But when the work turned to show the uploaded files, I just couldn't do it. Namely;
My filepond setup is as follows =>

`
$(function(){
    $.fn.filepond.registerPlugin(FilePondPluginImageResize, FilePondPluginImageTransform, FilePondPluginImagePreview, FilePondPluginFileRename, FilePondPluginFilePoster);

    $('#product_images').filepond({
        allowMultiple: true,
        server: {
	        process: 'imgupload',
            load: 'imguploaded/',
	    },
        files : [
            {
                source : $('#product_id').val(),
                options: {
                    type: 'local'
                }
            },
            {
                source : $('#product_id').val(),
                options: {
                    type: 'local'
                }
            }
        ],
        labelIdle: 'Seçebilir veya sürükle-bırak yapabilirsiniz. <span class="filepond--label-action">Gözat</span>',  
        allowImagePreview: true,
        allowImageResize: true,
        imageResizeMode: 'cover',
        imageResizeTargetWidth: 200,
        imageResizeTargetHeight: 200,
        fileRenameFunction: (file) => {
            return uuidv4()+`${file.extension}`;
        }
    });
</script>
`

- I use 2 filepond objects on the same page. That's why setoptions don't work
- My code to pull files from the server is as follows

` public function imguploadedAction()
    {
        $response = new Response();
        $contents = file_get_contents('files/products/images/1b9e3b5c-d86f-48c4-94b5-d224dc2182a9.jpg');
        $response
        ->setContent($contents)
        ->setContentType('image/jpeg')
        ->setHeader(
            'Content-Disposition', 
            "filename='deneme.jpg'", 
        )->send();
        return json_encode($response);
    }`

And it looks like this on the page.
![filepond](https://user-images.githubusercontent.com/504973/118345102-08301980-b53b-11eb-8164-239655594ec1.png)
My problem is that I cannot duplicate the file object since I do not know how many images have been loaded for this id before. 
I tried doing it with Ajax. I tried it with php, but it never happened.
my sample code for ajax
`
public function imgcountAction()
    {
        $this->view->disable();
        $request = new Request();
        $url = new Url();
        $product_id = $request->get('product_id');
        $resimler = ProductsImages::findByProductId($product_id);
        $resimpath = [];
        foreach ($resimler as $resim) {
            $detay = new \stdClass();
            $detay->source = $resim->image_path;
            $detay->options = array('type' => 'locale');
            $resimpath[] = $detay;
        }
        echo json_encode($resimpath);
        //echo json_encode("header('Content-Disposition: inline; filename=my-file.jpg');");
    }
`

`
function deneme(){ 
      $.ajax({
        type:'POST',
        url: "/yonetim/urunler/imgcount",
        data:{'product_id':$('#product_id').val()},
        success : function(data){
            return JSON.parse(data);
        }
        });   
    }
`

I am looking forward to your help to overcome this problem.

**Comment by rikschennink** [comment]
We require the issue template to be filled out on all new issues. The issue template helps us collect all the information we need to address your submission efficiently.

Your issue can be classified as an integration question or a question about specific features of FilePond we ask you kindly to submit it on [Stack Overflow](https://stackoverflow.com/questions/tagged/filepond).

**Please leave a comment after you've updated the issue.**
